src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -259,10 +259,11 @@ class UnconnectedCommunityTemplatesIndex extends Component<Props, State> {
   }
 
   private handleTemplateChange = event => {
-    const validationMessage = validateTemplateURL(event.target.value)
+    const trimmedValue = event.target.value.trim()
```

Comment:
naming nit: `value` is a vague name. `trimmedUrl` is much more self-documenting.